testing/trino-tests/src/test/java/io/trino/execution/TestConnectorEventListener.java

```diff
@@ -47,6 +47,7 @@ public void setUp()
                 .setSchema("tiny")
                 .setClientInfo("{\"clientVersion\":\"testVersion\"}")
                 .build();
+        DistributedQueryRunner.setupLogging();
```

Comment:
Do this in the constructor instead of putting this in all the tests.

Reply:
Done!